README.md

```diff
@@ -100,3 +100,61 @@ google_compute_instance.app.1:
+-   `var_file`: (Optional) Set variables in the Terraform configuration from a file. Path is relative to `dir`.
+
+The `apply` plan will run the `terraform_apply` task against the `localhost` target and optionally return the result of the `terraform_output` task. It accepts several fields:
+
+-   `dir`: (Optional) Path to Terraform project directory. Path is relative to CWD, unless an absolute path is specified.
+-   `state`: (Optional) Path to read and save state. Defaults to `terraform.tfstate`. Path is relative to `dir`
+-   `target`: (Optional) Resource to target. Operation will be limited to this resource and its dependencies. Accepts a single resource string or an array of resources.
+-   `var`: (Optional) Set Terraform variables, expects a hash with key value pairs representing variables and values.
+-   `var_file`: (Optional) Set variables in the Terraform configuration from a file. Path is relative to `dir`.
+-   `return_output`: (Optional) Return the result of the `terraform_output` task (defualts to `false`). 
+
+The `terraform_output` task will return the result of executing `terraform output`. It accepts several fields:
+
+-   `dir`: (Optional) Path to Terraform project directory. Path is relative to CWD, unless an absolute path is specified.
+-   `state`: (Optional) Path to read and save state. Defaults to `terraform.tfstate`. Path is relative to `dir`.
+
+## Destroying resources
+
+The `terraform_destroy` task will destroy resources and return the logs printed to stdout. It accepts several fields:
+
+-   `dir`: (Optional) Path to Terraform project directory. Path is relative to CWD, unless an absolute path is specified.
+-   `state`: (Optional) Path to read and save state. Defaults to `terraform.tfstate`. Path is relative to `dir`.
+-   `target`: (Optional) Resource to target. Operation will be limited to this resource and its dependencies. Accepts a single resource string or an array of resources.
+-   `var`: (Optional) Set Terraform variables, expects a hash with key value pairs representing variables and values.
+-   `var_file`: (Optional) Set variables in the Terraform configuration from a file. Path is relative to `dir`.
+
+The `destroy` plan will run the `terraform_destroy` task against the `localhost` and return it's result. It accepts several fields:
+
+-   `dir`: (Optional) Path to Terraform project directory. Path is relative to CWD, unless an absolute path is specified.
+-   `state`: (Optional) Path to read and save state. Defaults to `terraform.tfstate`. Path is relative to `dir`
+-   `target`: (Optional) Resource to target. Operation will be limited to this resource and its dependencies. Accepts a single resource string or an array of resources.
+-   `var`: (Optional) Set Terraform variables, expects a hash with key value pairs representing variables and values.
+-   `var_file`: (Optional) Set variables in the Terraform configuration from a file. Path is relative to `dir`.
+
```

Comment:
This might be a good place to add a quick note about `'` in the vars hash not working?